conanfile.py

```diff
@@ -1,38 +1,121 @@
 from conan import ConanFile
-from conan.tools.cmake import CMakeToolchain, CMakeDeps
-
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake, cmake_layout
+from conan.tools import files
+from conans import tools
+from conan.errors import ConanInvalidConfiguration
 
 required_conan_version = ">=1.46.2"
 
 
-class CuraBuildEnvironemtConan(ConanFile):
-    name = "cura-build-environment"
-    description = "Building Cura dependencies"
-    topics = ("conan", "python", "pypi", "pip")
-    settings = "os", "compiler", "build_type", "arch"
+class CuraEngineConan(ConanFile):
+    name = "curaengine"
+    license = "AGPL-3.0"
+    author = "Ultimaker B.V."
+    url = "https://github.com/Ultimaker/CuraEngine"
+    description = "Powerful, fast and robust engine for converting 3D models into g-code instructions for 3D printers. It is part of the larger open source project Cura."
+    topics = ("conan", "cura", "protobuf", "gcode", "c++", "curaengine", "libarcus", "gcode-generation")
```

Comment:
```suggestion
    topics = ("cura", "protobuf", "gcode", "c++", "curaengine", "libarcus", "gcode-generation")
```
nitpicking, the engine isn't about conan right? 

ALso, we might want to add 3D-printing as topic?